autode/mol_graphs.py

```diff
@@ -487,17 +511,17 @@ def is_isomorphic(graph1, graph2, ignore_active_bonds=False):
         return False
 
     # Always match on atom types
-    node_match = isomorphism.categorical_node_match('atom_label', 'C')
+    node_matcher = graph1.node_matcher
 
     if ignore_active_bonds:
         gm = isomorphism.GraphMatcher(graph1, graph2,
```

Comment:
Not sure how much I like 'gm' as a variable rather than something more explicit but it's still fairly obvious if you're reading the code

Reply:
`graph_matcher` is a function 😢